pingcap/tidb-operator
# tiflash: remove proxy log container and print proxy log to tiflash container

### What problem does this PR solve? 
close https://github.com/pingcap/tidb-operator/issues/3336

### What is changed and how does it work?
Remove `proxylog` container for `tlflash` pod and don't set `config.CommonConfig.Flash.FlashProxy.LogFile`.

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
`tiflash` pod can print logs out correctly.
```log
 kubectl logs basic-tiflash-0 -n demo -c tiflash
Logging information to /data0/logs/server.log
Logging errors to /data0/logs/error.log
[2020/10/09 11:37:28.402 +00:00] [INFO] [lib.rs:35] ["Welcome To TiFlash Raft Proxy"]
[2020/10/09 11:37:28.402 +00:00] [INFO] [lib.rs:37] ["Git Commit Hash:   6140a63a65643e65c074a8957fec0993513f6c19"]
[2020/10/09 11:37:28.402 +00:00] [INFO] [lib.rs:37] ["Git Commit Branch: HEAD"]
[2020/10/09 11:37:28.402 +00:00] [INFO] [lib.rs:37] ["UTC Build Time:    2020-09-25 12:53:06"]
[2020/10/09 11:37:28.403 +00:00] [INFO] [lib.rs:37] ["Rust Version:      rustc 1.46.0-nightly (16957bd4d 2020-06-30)"]
[2020/10/09 11:37:28.403 +00:00] [INFO] [lib.rs:37] ["Storage Engine:    tiflash"]
[2020/10/09 11:37:28.403 +00:00] [INFO] [lib.rs:37] ["Prometheus Prefix: tiflash_proxy_"]
[2020/10/09 11:37:28.403 +00:00] [INFO] [lib.rs:37] ["Profile:           release"]
[2020/10/09 11:37:28.409 +00:00] [INFO] [mod.rs:46] ["memory limit in bytes: 4138921984, cpu cores quota: 6"]
[2020/10/09 11:37:28.414 +00:00] [WARN] [server.rs:944] ["check: kernel"] [err="kernel parameters net.core.somaxconn got 128, expect 32768"]
```

Code changes

 - Has Go code change

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
Print proxy log of TiFlash to stdout
```
